test/pummel/test-tls-server-large-request.js

```diff
@@ -59,9 +59,9 @@ const server = tls.Server(options, common.mustCall(function(socket) {
   socket.pipe(mediator);
 }));
 
-server.listen(common.PORT, common.mustCall(function() {
+server.listen(0, common.mustCall(function() {
```

Comment:
Nit: while we are at it =)
```suggestion
server.listen(0, common.mustCall(() => {
```